### DISCUSSION TITLE
динамическое добавление агентов в очередь

### DISCUSSION THREAD

**Discussion by radioteh** [mikopbx/Core#59] [Ideas (Идеи)]
Добрый день. На текущий момент не получается создать очередь без добавления статических агентов в веб-интерфейсе. 
Будет ли реализован функционал динамического добавления (регистрации) агентов в очереди?
Будет ли реализован механизм приоритетов у агентов в очереди?

**Comment by boffart** [comment]
Для 2020.1.124 есть вот такая статья:
Динамические агенты очередей
https://wiki.mikopbx.com/faq:queue_dynamic_agents

**Comment by radioteh** [comment]
да, но при этом невозможно создать пустую очередь, для динамического добавления агентов:
![изображение](https://user-images.githubusercontent.com/50371750/101909058-09e16400-3bce-11eb-9845-40d9ac025f64.png)
и из-за того, что агенту нельзя назначить приоритет (parametr priority у статического агента), добавленный статический агент будет первым в очереди :(

**Reply by YanchikAmb** [reply]
если добавленного статического агента динамически убрать, то все работает как надо
![изображение](https://user-images.githubusercontent.com/9751378/107639952-a71a5080-6c82-11eb-9201-19b62d3e64c3.png)

**Reply by boffart** [reply]
хм.. по сути это и есть решение) 
над ограничением с пустой очередью подумаем.
